docs/quickstart.mdx

```diff
@@ -88,13 +79,24 @@ Submit the form and TEST be on your way.
 
 This is a Sapphire tutorial and you're already a Solidity expert, so let's not
 bore you with explaining the gritty details of the contract.
-Start by pasting [Vigil.sol] into `contracts/Vigil.sol`.
+Start by pasting `Vigil.sol` into `contracts/Vigil.sol`.
+
+1. Create a new file called `Vigil.sol` under `contracts/`:
+2. Paste the following contract into it:
+
+<details>
+  <summary> Vigil.sol contract </summary>
+
+  ![code Solidity](../examples/hardhat/contracts/Vigil.sol)
+</details>
+
+3. Add deploy and check tasks to Hardhat config:
 
-While you're there, also place [run-vigil.ts] into `scripts/run-vigil.ts`.
-We'll need that later.
+<details>
+  <summary> hardhat.config.ts </summary>
 
-[Vigil.sol]: https://github.com/oasisprotocol/sapphire-paratime/blob/stable/clients/js/1.x/examples/hardhat/contracts/Vigil.sol
-[run-vigil.ts]: https://github.com/oasisprotocol/sapphire-paratime/blob/stable/clients/js/1.x/examples/hardhat/scripts/run-vigil.ts
+  ![code typescript](../examples/hardhat/hardhat.config.ts#config-tasks)
```

Comment:
Looks like [Netlify](https://app.netlify.com/sites/oasisprotocol-sapphire-paratime/deploys/67c07a1ef4a3320008214eb7) doesn't know about this file until it gets to `docs` 🤔

Reply:
I can split this into four PRs or just do the one shot bump.

Reply:
Merged latest sapphire-paratime commit into docs repo.

Reply:
Needs https://github.com/oasisprotocol/docs/pull/1160